Marlin/Configuration.h

```diff
@@ -1652,7 +1652,7 @@
 #endif
 
 /**
- * Printer Event LEDs
+ * Printer Event LEDs and LCD Control Menu
```

Comment:
"LCD Control Menu" should be changed to "LED Control Menu"

Reply:
Oops.